submarine-workbench/interpreter/spark-interpreter/README.md

```diff
@@ -17,13 +17,16 @@
 ## Test Submarine Spark Interpreter
 
 ### Execute test command
-```
-export SUBMARINE_HOME=/path/to/your/submarine_home
-java -jar spark-interpreter-0.3.0-SNAPSHOT-shade.jar spark spark-interpreter-id test
+```bash
+# spark interpreter
+java -jar spark-interpreter-{{version}}-shade.jar spark spark-interpreter-id test
```

Comment:
Now, `spark-interpreter-{{version}}-shade.jar` jar package name already change to `submarine-spark-interpreter-{{version}}-shade.jar`